pennylane/templates/subroutines/approx_time_evolution.py

```diff
@@ -24,6 +24,12 @@ class ApproxTimeEvolution(Operation):
     r"""Applies the Trotterized time-evolution operator for an arbitrary Hamiltonian, expressed in terms
     of Pauli gates.
 
+        .. note::
+
+            This operation behaves identically to :class:`~pennylane.TrotterProduct` and will be deprecated in
```

Comment:
Maybe "is a special case of"?

Reply:
Are we merging this after TrotterProduct so we can confirm the links work?

Reply:
Yes!

Reply:
Added